kubetest/gke.go

```diff
@@ -235,8 +235,12 @@ func newGKE(provider, project, zone, region, network, image, cluster string, tes
 	if *upgradeArgs != "" {
 		fields, val, exist := extractField(strings.Fields(*upgradeArgs), "--upgrade-target")
 		if exist {
-			if val == "gke-latest" {
-				if val, err = getLatestGKEVersion(project, zone); err != nil {
+			if strings.HasPrefix(val, "gke-latest") {
```

Comment:
Is the format of the `--upgrade-target` documented?

Reply:
added reference from main repo where the upgrade target is actually used

Reply:
IIUC, the linked file takes the output of the parsing here. I care more about what the valid input value. It would be great if you can say something like `gke-latest will be resolved to the latest gke version, gke-latest-v1.7 will be resolved to the latest 1.7 patch version supported on gke`.

Reply:
Feel free to apply the lgtm yourself after addressing this comment.

Reply:
done, added more comments